diesel_cli/src/print_schema.rs

```diff
@@ -7,6 +7,8 @@ use std::error::Error;
 use std::fmt::{self, Display, Formatter, Write};
 use std::io::Write as IoWrite;
 
+const SCHEMA_HEADER: &str = "// @generated automatically by Diesel CLI.\r\n\r\n";
```

Comment:
I don't think `\r\n` is reasonable here. Should use `writeln!` or just use `\n` like other places?

Reply:
removed additional carriage return char